commons/src/main/java/com/powsybl/commons/io/table/AbstractTableFormatter.java

```diff
@@ -31,13 +31,33 @@ protected AbstractTableFormatter(Writer writer, TableFormatterConfig config, Col
         this.column = 0;
     }
 
+    public AbstractTableFormatter(Writer writer, TableFormatterConfig config, int length) {
+        this.writer = Objects.requireNonNull(writer);
+        this.config = Objects.requireNonNull(config);
+        this.column = 0;
+        columns = new Column[length];
+        for (int i = 0; i < length; i++) {
+            columns[i] = new Column(String.valueOf(i));
+        }
+    }
```

Comment:
Is probably useless: tables should always define the columns

Reply:
we have to do this 'cause we need a 'columns' table containing non null objects for some implemented functions.